ui/public/config.json

```diff
@@ -46,7 +46,119 @@
+    "adminguide/virtual_machines.html#resetting-ssh-keys": "adminguide/virtual_machines.html#resetting-ssh-keys",
+    "adminguide/virtual_machines.html#restoring-vm-backups": "adminguide/virtual_machines.html#restoring-vm-backups",
+    "adminguide/virtual_machines.html#stopping-and-starting-vms": "adminguide/virtual_machines.html#stopping-and-starting-vms",
+    "adminguide/virtual_machines.html#using-ssh-keys-for-authentication": "adminguide/virtual_machines.html#using-ssh-keys-for-authentication",
+    "adminguide/virtual_machines.html#virtual-machine-snapshots": "adminguide/virtual_machines.html#virtual-machine-snapshots",
+    "installguide/configuration.html#adding-a-pod": "installguide/configuration.html#adding-a-pod",
+    "installguide/configuration.html#adding-a-zone": "installguide/configuration.html#adding-a-zone",
+    "installguide/configuration.html#add-primary-storage": "installguide/configuration.html#add-primary-storage",
+    "installguide/configuration.html#add-secondary-storage": "installguide/configuration.html#add-secondary-storage",
+    "plugins/cloudian-connector.html": "plugins/cloudian-connector.html",
+    "plugins/cloudstack-kubernetes-service.html": "plugins/cloudstack-kubernetes-service.html",
+    "plugins/cloudstack-kubernetes-service.html#creating-a-new-kubernetes-cluster": "plugins/cloudstack-kubernetes-service.html#creating-a-new-kubernetes-cluster",
+    "plugins/cloudstack-kubernetes-service.html#deleting-kubernetes-cluster": "plugins/cloudstack-kubernetes-service.html#deleting-kubernetes-cluster",
+    "plugins/cloudstack-kubernetes-service.html#kubernetes-supported-versions": "plugins/cloudstack-kubernetes-service.html#kubernetes-supported-versions",
+    "plugins/cloudstack-kubernetes-service.html#scaling-kubernetes-cluster": "plugins/cloudstack-kubernetes-service.html#scaling-kubernetes-cluster",
+    "plugins/cloudstack-kubernetes-service.html#starting-a-stopped-kubernetes-cluster": "plugins/cloudstack-kubernetes-service.html#starting-a-stopped-kubernetes-cluster",
+    "plugins/cloudstack-kubernetes-service.html#stopping-kubernetes-cluster": "plugins/cloudstack-kubernetes-service.html#stopping-kubernetes-cluster",
+    "plugins/cloudstack-kubernetes-service.html#upgrading-kubernetes-cluster": "plugins/cloudstack-kubernetes-service.html#upgrading-kubernetes-cluster",
+    "plugins/nuage-plugin.html?#optional-create-and-enable-vpc-offering": "plugins/nuage-plugin.html?#optional-create-and-enable-vpc-offering",
+    "plugins/nuage-plugin.html?#vpc-offerings": "plugins/nuage-plugin.html?#vpc-offerings",
+    "plugins/quota.html": "plugins/quota.html",
+    "plugins/quota.html#quota-credits": "plugins/quota.html#quota-credits",
+    "plugins/quota.html#quota-tariff": "plugins/quota.html#quota-tariff"
+  },
   "plugins": [],
   "basicZoneEnabled": true
```

Comment:
@shwstppr do we need to document this new config? (I guess this is to disable the basic zone option in the zone deployment zone, which is only for root admins, it may even be removed cc @utchoang)

Additionally, does it make sense to move these configs above docHelp mappings? (as the list is huge, someone searching for these will have to scroll to the bottom) - what do you think?

Reply:
@rhtyd moved `docHelpMappings` at the end.
Also, added prebuild and postbuild scripts to add/remove all docHelp suffixes in the config file during the build process.